ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_service.proto

```diff
@@ -140,3 +143,16 @@ message GetLedgerEndResponse {
   LedgerOffset offset = 1;
 }
 
+message GetLatestPrunedOffsetsRequest {
+  // Empty for now, but may contain fields in the future.
+}
+
+message GetLatestPrunedOffsetsResponse {
+  // The offset up until the ledger has been pruned, without pruning all divulgence events.
+  LedgerOffset participant_pruned_up_to_inclusive = 1;
+
+  // The offset up until the ledger has been pruned including pruning all divulgence events.
+  // For more details about all divulgence events pruning,
+  // see PruneRequest.prune_all_divulged_contracts in ``participant_pruning_service.proto``.
```

Comment:
```suggestion
  // The offset up to which all divulged events have been pruned on the ledger. It can be at or before the
  // ``participant_pruned_up_to_inclusive`` offset.
  // For more details about all divulged events pruning,
  // see ``PruneRequest.prune_all_divulged_contracts`` in ``participant_pruning_service.proto``.
```